docs/features/github/tutorials/automated-pull-requests.mdx

```diff
@@ -143,7 +143,9 @@ To add the repository to Otterize Cloud, navigate to the [Integrations page](htt
 5. In the `Open PR on` section, select the `Repository` field, and provide the owner and organization names in the form: *<github user name\>/otterize-tutorial-github*.
 6. Select the `Base Branch` field, and enter *main*.
 7. Select the `ClientIntents path` field, and enter `intents`. It represents the relative path hosting the ClientIntents manifests.
-8. Next, click the `Add` button. This will redirect you to GitHub. If needed, select your GitHub account that owns the tutorial repository and click `continue`. Then, select the desired organization and the tutorial repository (*otterize-tutorial-github*), as depicted in the picture below. Finally, click on `Install`.
+8. You can set the pull request to be auto-merged, using GitHub's merge queue, by enabling the auto merge toggle.
```

Comment:
```suggestion
8. You can set the pull request to be auto-merged, by enabling the auto merge toggle.
```

I think this doesn't actually use merge queue, which is a separate GitHub Enterprise feature that queues up PRs to be merged, including rebasing each one in line to include all the others, so that PRs can go in quickly and have all previous code as part of their tests, to avoid cross-breakage (the case when all PRs pass tests, but when combined in main they break)